src/momento/config/transport/grpc_configuration.py

```diff
@@ -12,3 +13,11 @@ def get_deadline(self) -> timedelta:
     @abstractmethod
     def with_deadline(self, deadline: timedelta) -> GrpcConfiguration:
         pass
+
+    @abstractmethod
+    def get_root_certificates(self) -> Optional[bytes]:
+        pass
+
+    @abstractmethod
+    def with_root_certificates(self, certificate: bytes) -> GrpcConfiguration:
+        pass
```

Comment:
i don't really love adding more surface area to the configuration object APIs without a customer-facing use case. also isn't your local setup going to become obsolete as soon as the real service goes live, since it's not the same tech stack?

Reply:
I won't block the PR on it if you feel strongly about it, it's not an unreasonable thing to support; I'm just doing my usual advocacy for keeping API footprint as small as possible

Reply:
@malandis did you have any more thoughts on this in response to comments above?